docs/releases/0_9_13.rst

```diff
@@ -153,6 +154,7 @@ Graphite-web
 * Fix for SVG graphs (grobian, obfuscurity)
 * Skip empty target parameters (obfuscurity)
 * Remove unnecessary dependencies (obfuscurity)
+* Fixing RTD builds (obfuscurity)
```

Comment:
Yeah, about this. The 0.9.x build is **still** not working. The master build was fixed but that was all thanks to @brutasse. Either way, I don't think it warrants a mention in the Release Notes since it's technically related to infrastructure.